src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/config/Config.java

```diff
@@ -149,6 +158,8 @@
     private boolean restCodeReview;
     private boolean restVerified;
     private boolean gerritBuildCurrentPatchesOnly;
+    private boolean gerritAbortNewPatchsets;
```

Comment:
When upgrading from a previous version these fields will become ```false```, is that the intended behaviour?